eval.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+
+
+# echo =========== Generation Predictions for English images using UserVideo ==========
+# python generate_predictions.py
+
```

Comment:
can we move this code to a submodule?

Reply:
All the eval codes are refactored and moved into ocr_eval folder.